### ISSUE TITLE
Next release/best way of using last version

### ISSUE THREAD

**Issue by InigoReizabal** [plotly/dash-daq#162]
Hello. I'm currently using the last release of `dash-daq` repo but the release was done on 27 Apr 2020 and it does not contain some of bug fixes that I need. The question is:

- Considering that the last release was done more than one year an a half ago, are you planning to do a release any time soon?

- If not, how do you recommend me to install/use the latest available version?

Thank you very much for this great project.

**Comment by alexcjohnson** [comment]
Right, we do have a bunch of bug fixes completed but not released. It's just waiting for us to review https://github.com/plotly/dash-daq/pull/160 and complete a little more QA before we're ready to release. Hopefully won't be too long but I can't promise any particular timeline.

If you want to use an unreleased version, in this repo we still commit the builds so you can install directly from GitHub without needing to build https://pip.pypa.io/en/stable/topics/vcs-support/:
```
pip install git+https://github.com/plotly/dash-daq.git#egg=dash_daq
# or to include the unmerged PR 160
pip install git+https://github.com/Karan-S-Mittal/dash-daq.git@bug-fix#egg=dash_daq
```
(note you can include those strings, ie everything after `pip install`, in a requirements file as well)

**Comment by pmdias** [comment]
Just to clarify a small detail, considering that the `dash_daq/metadata.json` is [not commited](https://github.com/plotly/dash-daq/blob/master/.gitignore#L7) to version control, doesn't an installation from git require that the `npm run build` step be done locally, in order to install that file also?

These are the steps that I followed to successfully install from git

```bash
$ npm install
$ npm run build
$ python -m pip install .
```

Otherwise I get a `FileNotFoundError: [Errno 2] No such file or directory: 'lib/python3.9/site-packages/dash_daq/metadata.json'` when importing `dash_daq` in a Python script. Am I missing something on this?

**Comment by alexcjohnson** [comment]
Thanks @pmdias - you're right, looks like this project is in an old state that [still requires the metadata file at runtime](https://github.com/plotly/dash-daq/blob/27998477f4cb70af542dc18c91db981b6161112b/dash_daq/__init__.py#L15) whereas our other projects have moved away from that pattern, instead using [`._imports_`](https://github.com/plotly/dash-bio/blob/aab01c520460e69eda29e07c97c3a9f9aef61f60/dash_bio/__init__.py#L28-L29). So until that's cleaned up `npm i` and `npm run build` are required. Let's keep this issue open until we've resolved that.

**Comment by MrTeale** [comment]
I have created #167 which removes the use of `metadata.json`. To install using this method, use:
```shell
pip install git+https://github.com/plotly/dash-daq.git#egg=dash_daq
# or while #167 is open
pip install git+https://github.com/mrteale/dash-daq.git@fix/131-imports#egg=dash_daq
```

**Comment by Morganplant** [comment]
Any update on this? 
When can we expect next release?
Thanks!

**Comment by beginner000001** [comment]
Nowadays, mobile first is the standard, but unfortunately the Dash Knob control cannot be used on touchscreen devices. It's a shame because the Knob looks great and adds a lot of value to the application. It would be great if it could be made touch-compatible.
